addons/pos_restaurant/static/tests/tours/floor_screen_tour.js

```diff
@@ -148,15 +148,22 @@ registry.category("web_tour.tours").add("FloorScreenTour", {
 
             // Check the linking of tables
             FloorScreen.clickFloor("Main Floor"),
+            FloorScreen.clickTable("4"),
+            ProductScreen.clickDisplayedProduct("Coca-Cola"),
+            ProductScreen.back(),
             FloorScreen.linkTables("5", "4"),
             FloorScreen.isChildTable("5"),
             Utils.refresh(),
             FloorScreen.isChildTable("5"),
 
             // Check that tables are unlinked automatically when the order is done
             FloorScreen.clickTable("5"),
-            ProductScreen.tableNameShown("4"),
-            ProductScreen.clickDisplayedProduct("Coca-Cola"),
+            ProductScreen.tableNameShown("4 & 5"),
+            ProductScreen.selectedOrderlineHas("Coca-Cola", "1.0"),
+            ProductScreen.back(),
+            FloorScreen.goTo("5"),
```

Comment:
```suggestion
            FloorScreen.isShown(),
            FloorScreen.goTo("5"),
```

before proceeding to go table 5, check whether the floor screen is shown.